spec/models/authentication_spec.rb

```diff
@@ -1,22 +1,22 @@
-RSpec.describe Authentication do
-  let(:auth) { build(:authentication) }
+# RSpec.describe Authentication do
+#   let(:auth) { build(:authentication) }
 
-  context "encrypt protected data" do
-    before do
-      auth.secret = "blah"
-      auth.save
-    end
-    it "encrypts the secret field in the encryptions table" do
-      results = ActiveRecord::Base.connection.exec_query("select secret from encryptions")
-      rows = results.instance_variable_get(:@rows)[0][0]
-      expect(rows).to match(/v2/)
-      expect(rows).not_to match(/blah/)
-    end
+#   context "encrypt protected data" do
+#     before do
+#       auth.secret = "blah"
+#       auth.save
+#     end
+#     it "encrypts the secret field in the encryptions table" do
+#       results = ActiveRecord::Base.connection.exec_query("select secret from encryptions")
+#       rows = results.instance_variable_get(:@rows)[0][0]
+#       expect(rows).to match(/v2/)
+#       expect(rows).not_to match(/blah/)
+#     end
 
-    it "returns the unencrypted version to the caller" do
-      auth.reload
-      expect(auth.secret).to eq "blah"
-      expect(auth.secret).not_to match(/v2/)
-    end
-  end
-end
+#     it "returns the unencrypted version to the caller" do
+#       auth.reload
+#       expect(auth.secret).to eq "blah"
+#       expect(auth.secret).not_to match(/v2/)
+#     end
+#   end
+# end
```

Comment:
I'm assuming this is temporary until we can figure out how to have testable migrations?

Reply:
Correct

Reply:
My PR got this working, and I think it's pretty straightforward, but I'm ok merging as is, and then fixing this in a follow up PR.